controllers/components/kueue/kueue_support.go

```diff
@@ -33,3 +33,11 @@ func manifestsPath() odhtypes.ManifestInfo {
 		SourcePath: "rhoai",
 	}
 }
+
+func extramanifestsPath() odhtypes.ManifestInfo {
+	return odhtypes.ManifestInfo{
+		Path:       odhdeploy.DefaultManifestPath,
+		ContextDir: ComponentName,
+		SourcePath: "rhoai/ocp-4.17-addons",
```

Comment:
is the `ocp-4.17-addons` going to be kept forever ? or it is expected to be changed over time i.e. `ocp-4.18-addons` ?